db/migrations/20200310230012_create_medianizer_storage_table.sql

```diff
@@ -0,0 +1,46 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+CREATE TABLE maker.medianizer_val
```

Comment:
I think we probably want an `address_id` in these tables, since there are multiple `Median` contracts.